examples/rfq_quickstart/main.rs

```diff
@@ -0,0 +1,201 @@
+        }
+
+        self
+    }
+}
+
+#[tokio::main]
+async fn main() {
+    tracing_subscriber::fmt()
+        .with_env_filter(EnvFilter::from_default_env())
+        .with_target(false)
+        .init();
+
+    let cli = Cli::parse().with_defaults();
+
+    let chain = Chain::from_str(&cli.chain)
+        .unwrap_or_else(|_| panic!("Unknown chain {chain}", chain = cli.chain));
+
+    let tycho_url = env::var("TYCHO_URL").unwrap_or_else(|_| {
+        get_default_url(&chain)
+            .unwrap_or_else(|| panic!("Unknown URL for chain {chain}", chain = cli.chain))
+    });
+
+    let tycho_api_key: String =
+        env::var("TYCHO_API_KEY").unwrap_or_else(|_| "sampletoken".to_string());
+
+    // Get Bebop WebSocket credentials
+    let bebop_ws_user = env::var("BEBOP_WS_USER")
+        .expect("BEBOP_WS_USER environment variable is required. Contact Bebop for credentials.");
+    let bebop_ws_key = env::var("BEBOP_WS_KEY")
+        .expect("BEBOP_WS_KEY environment variable is required. Contact Bebop for credentials.");
+
+    println!("Loading tokens from Tycho... {url}", url = tycho_url.as_str());
+    let all_tokens =
+        load_all_tokens(tycho_url.as_str(), false, Some(tycho_api_key.as_str()), chain, None, None)
+            .await;
+    println!("Tokens loaded: {num}", num = all_tokens.len());
+
+    let sell_token_address = Bytes::from_str(
+        &cli.sell_token
+            .expect("Sell token not provided"),
+    )
+    .expect("Invalid address for sell token");
+    let buy_token_address = Bytes::from_str(
+        &cli.buy_token
+            .expect("Buy token not provided"),
+    )
+    .expect("Invalid address for buy token");
+    let sell_token = all_tokens
+        .get(&sell_token_address)
+        .expect("Sell token not found")
+        .clone();
+    let buy_token = all_tokens
+        .get(&buy_token_address)
+        .expect("Buy token not found")
+        .clone();
+    let amount_in =
+        BigUint::from((cli.sell_amount * 10f64.powi(sell_token.decimals as i32)) as u128);
+
+    println!(
+        "Looking for RFQ quotes for {amount} {sell_symbol} -> {buy_symbol} on {chain:?}",
+        amount = cli.sell_amount,
+        sell_symbol = sell_token.symbol,
+        buy_symbol = buy_token.symbol
+    );
+
+    // Set up Bebop RFQ client
+    let mut bebop_pairs = HashSet::new();
+    bebop_pairs.insert((sell_token_address.to_string(), buy_token_address.to_string()));
+    bebop_pairs.insert((buy_token_address.to_string(), sell_token_address.to_string()));
+
+    let mut bebop_quote_tokens = HashSet::new();
+    bebop_quote_tokens.insert(buy_token_address.to_string());
+
+    println!("Connecting to Bebop RFQ WebSocket...");
+    let bebop_client = BebopClient::new(
+        chain.into(), // Convert from models::Chain to dto::Chain
+        bebop_pairs,
+        cli.tvl_threshold,
+        bebop_ws_user,
+        bebop_ws_key,
+        bebop_quote_tokens,
+    )
+    .expect("Failed to create Bebop client");
```

Comment:
I'm thinking that we should make a builder for BebopClient 🤔 
This way we can have the defaults for the quote tokens for example (I think it's weird that the user needs to choose these)
Also the chain in the BebopClient should be the models one, not the dto one

Reply:
and what if they don't pass any `bebop_pairs`? 🤔 could that just mean that they want all?

Reply:
(Implemented the builder pattern)

Reply:
Yeah @dianacarvalho1